### PR TITLE
Mobile menu feature

### PR DESCRIPTION
**Pull request by Drew246**
Added functionality to the mobile menu button. Navigation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,46 +29,64 @@
...
-          </li>
-          <li>
-            <a href="#"><i class="fa fa-medium"></i></a>
+        <a href="#" class="mobile-menu" onclick="toggleMobileMenu()">
+          <i class="fa fa-bars"></i>
+        </a>
+        <ul class="mobile-menu-list" id="mobileMenuList">
+          <li class="nav-item">
+            <a href="#portfolio" class="nav-link" onclick="closeMobileMenu()"
+              >Portfolio</a
```
Comment (AmaduKamara) [inline index.html:38]: - [ ] The mobile menu is not displaying as it should. It shows in a little box window when the hamburger menu is clicked. It actually closes when the X icon is clicked. Kindly take note of a few points when fixing the current issue:
- The mobile should stay fixed and not scroll when the user scrolls up. 👍🏽
- It should be full in width and height of the screen 👍🏽
- The menu items' styles should match the given design 👍🏽

Screenshot 2023-07-25 at 2 59 53 PM

**Review by AmaduKamara — APPROVED** [review]
# Status: Required Changes 🟢 🟢 🟢

Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used HTML and CSS Linters as required 👍🏽 
✅ Linter checks passed successfully 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (Drew246) [reply index.html:38]: Thank you for the feedback. I was aware that other changes needed to be done but I really wanted a review of my current progress so I submitted.

**Review by danmainah — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️  Implemented the change requested to ensure the menu design matches the design given.
✔️  Implemented the changes requested to ensure the user experience is as expected, the position of the menu is fixed even when scrolling.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**